inference-engine/src/vpu/graph_transformer/src/model/data.cpp

```diff
@@ -216,14 +216,15 @@ void DataNode::serializeIOInfo(BlobSerializer& serializer) const {
     auto ioBufferOffset = attrs().get<int>("ioBufferOffset");
     serializer.append(checked_cast<uint32_t>(ioBufferOffset));
 
-    auto nameLength = checked_cast<uint32_t>(_name.length());
-    auto nameLengthAligned = alignVal(nameLength, 16u);
+    auto nameLength = checked_cast<uint32_t>(_name.size());
```

Comment:
Why not to leave it with `length()`?

Reply:
In general, it does not matter. But you are right.
Reverted